src/components/Footer.tsx

```diff
@@ -14,19 +14,19 @@ export default function Footer() {
         {/* Right: Social Icons */}
         <div className="flex space-x-4 text-brandPink text-xl">
           <a
-            href="https://www.instagram.com"
+            href="https://www.instagram.com/dalyradianceaesthetics?igsh=ODZ2NTUzM2g4bWly&utm_source=qr"
```

Comment:
Just a suggestion, but you might want to put social links in a `constants.ts` file and import them from there. Just makes it easier to make sure all the links get updated where they should in future. E.g.:

```
export const SOCIAL_LINKS = {
  INSTAGRAM: "https://instagram.com",
  FACEBOOK: "https://facebook.com"
};
```

then `import {SOCIAL_LINKS} from './constants.ts'` so you can use `SOCIAL_LINKS.INSTAGRAM`

Up to you though, doesn't block this being merged.